src/services/enum-utilities.ts

```diff
@@ -138,18 +133,43 @@ export default class EnumUtilities {
   }
 
   /** Returns True if filing is an Administrative Dissolution. */
-  static isTypeAdministrativeDissolution (item: any): boolean {
-    return (item.name === FilingTypes.DISSOLUTION && item.dissolutionType === DissolutionTypes.ADMINISTRATIVE)
+  static isTypeDissolutionAdministrative (item: any): boolean {
+    return (
+      // the property in a todo item or filing item:
+      item.filingSubType === FilingSubTypes.DISSOLUTION_ADMINISTRATIVE ||
+      // the property in a state filing:
+      item.dissolution?.dissolutionType === FilingSubTypes.DISSOLUTION_ADMINISTRATIVE
+    )
```

Comment:
I realized this method, and the ones below, are called from different components using different data objects. So, I added the comments indicating the items the method supports, and the code checks all possibilities.

I don't like this, but fixing it would require some API changes (since the API doesn't provide filing sub-type consistently) and maybe splitting this method into separate cases for API objects or UI objects. I think this is OK for now but I admit it's fragile (*).

(*) The purpose of these tickets/PRs is to implement some new stuff, not refactor all the old stuff. The code changes above and below are actually fixes -- I found a few broken scenarios that now work.